be1-go/docs/README.md

```diff
@@ -36,37 +36,36 @@ The project is organized into different modules as follows
 
 ```
 .
-├── channel             # contains the abstract definition of a channel
-│   ├── authentication  # channel implementation for an authentication channel
-│   ├── chirp           # channel implementation for a chirp channel
-│   ├── coin            # channel implementation for a coin channel
-│   ├── consensus       # channel implementation for a consensus channel
-│   ├── election        # channel implementation for an election channel
-│   ├── generalChirping # channel implementation for a universal post channel
-│   ├── lao             # channel implementation for a LAO channel
-│   ├── reaction        # channel implementation for a reaction channel
-│   └── registry        # helper for registry
-├── cli                 # command line interface
-├── crypto              # defines the cryptographic suite 
+├── channel                   # contains the abstract definition of a channel NEED TO BE DELETED
+├── cli                       # command line interface
+├── crypto                    # defines the cryptographic suite
 ├── docs
-├── hub                 # contains the abstract definition of a hub
-│   ├── standard_hub    # hub implementation 
-├── inbox               # helper to store messages used by channels
-├── message             # message types and marshaling/unmarshaling logic
-├── network             # module to set up Websocket connections
-│   └── socket          # module to send/receive data over the wire
-├── popcha              # HTTP server and back-end logic for PoPCHA
-└── validation          # module to validate incoming/outgoing messages
+├── hub                       # contains the abstract definition of a hub NEED TO BE DELETED
+├── inbox                     # helper to store messages used by channels NEED TO BE DELETED
+├── internal                  
```

Comment:
why move all the app logic into `internal` ? I don't understand the logic behind the design decision 🤔

Reply:
I tried to follow [this doc](https://go.dev/doc/modules/layout) on the go site about module organization and [this article](https://eli.thegreenplace.net/2019/simple-go-project-layout-with-modules/). Inside the part _Package or command with supporting packages_,they advice to put everything inside the internal package.

Reply:
Okay, so what you're saying is that you still expect all the packages to move around and hence to keep everything in internal for now so as to avoid breaking changes for third party developers.

I don't think having other libraries depending on our PoP implementation is too much of a risk for now.... but okay, sure :-)